packages/core-app-api/config.d.ts

```diff
@@ -114,6 +114,11 @@ export interface Config {
      * @visibility frontend
      */
     environment?: string;
+    /**
+     * The 'allowGuestMode' flag for GuestUserIdentity.
+     * @visibility frontend
+     */
+    allowGuestMode?: boolean;
```

Comment:
Hmm i would have to explore this some more. This addition worked in my case but if there is a different way of doing this I can definitely change it. So far auth-backend has auth-common as a dependency. Which package would represent the "front-end"? i assumed that was core-app-api.

Reply:
Yeah it's just the case of finding a good home for it. Where it lies now, in a frontend-only package, it's not clear how a backend deployment would be guaranteed to have this package in its node_modules so that its config definition is present and loadable.